bitnami/thanos/templates/alert-rule/compaction.yml

```
@@ -10,7 +10,7 @@ Generated from https://github.com/thanos-io/thanos/blob/main/examples/alerts/ale
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
-  name: {{ template "common.names.fullname" . }}-compact
+  name: {{ template "common.names.fullname" . }}
```

One small change, keep the PrometheusRule name as it is to avoid conflicts with [prometheusrule.yaml](https://github.com/bitnami/charts/blob/main/bitnami/thanos/templates/prometheusrule.yaml)
```suggestion
  name: {{ template "common.names.fullname" . }}-compact
```

---

You are right, I missed this one. Thanks